packages/rainbowkit/src/wallets/walletConnectors/injectedWallet/injectedWallet.ts

```diff
@@ -21,7 +21,8 @@ export const injectedWallet = ({
       wallet =>
         wallet.installed &&
         (wallet.connector instanceof InjectedConnector ||
```

Comment:
This should catch the Rainbow connector, so we'll have to make changes to the `installed` logic in that connector instead

Reply:
yeah just pushed missed the param